adam-core/src/test/scala/org/bdgenomics/adam/rdd/features/FeatureRDDSuite.scala

```diff
@@ -60,240 +61,360 @@ class FeatureRDDFunctionsSuite extends ADAMFunSuite with TypeCheckedTripleEquals
-    val inputPath = resourcePath("features/Homo_sapiens.GRCh37.75.trun100.gtf")
+    val inputPath = resourcePath("Homo_sapiens.GRCh37.75.trun100.gtf")
     val features = sc.loadGtf(inputPath)
     val outputPath = tempLocation(".narrowPeak")
     features.saveAsNarrowPeak(outputPath)
   }
 
   sparkTest("round trip GTF format") {
-    val inputPath = resourcePath("features/Homo_sapiens.GRCh37.75.trun100.gtf")
+    val inputPath = resourcePath("Homo_sapiens.GRCh37.75.trun100.gtf")
     val expected = sc.loadGtf(inputPath)
     val outputPath = tempLocation(".gtf")
     expected.saveAsGtf(outputPath)
 
     // grab all partitions, may not necessarily be in order; sort by reference
     val actual = sc.loadGtf(outputPath + "/part-*")
-    val pairs = expected.coalesce(1).sortByReference().zip(actual.coalesce(1).sortByReference()).collect
+    val pairs = expected.transform(_.coalesce(1)).sortByReference().rdd.zip(actual.transform(_.coalesce(1)).sortByReference().rdd).collect
 
     // separate foreach since assert is not serializable
     pairs.foreach({ pair: (Feature, Feature) => assert(pair._1 === pair._2) })
   }
 
   sparkTest("save GFF3 as GTF format") {
-    val inputPath = resourcePath("features/dvl1.200.gff3")
+    val inputPath = resourcePath("dvl1.200.gff3")
     val features = sc.loadGff3(inputPath)
     val outputPath = tempLocation(".gtf")
     features.saveAsGtf(outputPath)
   }
 
   sparkTest("save GFF3 as BED format") {
-    val inputPath = resourcePath("features/dvl1.200.gff3")
+    val inputPath = resourcePath("dvl1.200.gff3")
     val features = sc.loadGff3(inputPath)
     val outputPath = tempLocation(".bed")
     features.saveAsBed(outputPath)
   }
 
   sparkTest("save GFF3 as IntervalList format") {
-    val inputPath = resourcePath("features/dvl1.200.gff3")
+    val inputPath = resourcePath("dvl1.200.gff3")
     val features = sc.loadGff3(inputPath)
     val outputPath = tempLocation(".interval_list")
     features.saveAsIntervalList(outputPath)
   }
 
   sparkTest("save GFF3 as NarrowPeak format") {
-    val inputPath = resourcePath("features/dvl1.200.gff3")
+    val inputPath = resourcePath("dvl1.200.gff3")
     val features = sc.loadGff3(inputPath)
     val outputPath = tempLocation(".narrowPeak")
     features.saveAsNarrowPeak(outputPath)
   }
 
   sparkTest("round trip GFF3 format") {
-    val inputPath = resourcePath("features/dvl1.200.gff3")
+    val inputPath = resourcePath("dvl1.200.gff3")
     val expected = sc.loadGff3(inputPath)
     val outputPath = tempLocation(".gff3")
     expected.saveAsGff3(outputPath)
 
     // grab all partitions, may not necessarily be in order; sort by reference
     val actual = sc.loadGff3(outputPath + "/part-*")
-    val pairs = expected.coalesce(1).sortByReference().zip(actual.coalesce(1).sortByReference()).collect
+    val pairs = expected.transform(_.coalesce(1)).sortByReference().rdd.zip(actual.transform(_.coalesce(1)).sortByReference().rdd).collect
```

Comment:
eesh, this line keeps getting longer!  I wish I could've separated it out into a utility method or class, but I couldn't get that to work due to serialization issues.  If you would like to take a shot at it, go fer it!